rust/processor/src/db/common/models/default_models/transactions.rs

```diff
@@ -195,20 +195,34 @@ impl Transaction {
 
         match txn_data {
             TxnData::User(user_txn) => {
-                let payload = user_txn
+                let request = &user_txn
                     .request
                     .as_ref()
-                    .expect("Getting user request failed.")
+                    .expect("Getting user request failed.");
+
+                let mut payload_cleaned = None;
+                let (payload, payload_type) = match request.payload.as_ref() {
+                    Some(payload) => {
+                        payload_cleaned = get_clean_payload(&payload, version);
+                        (payload_cleaned, Some(get_payload_type(&payload)))
+                    },
+                    None => (None, None),
+                };
+
+                let (payload_cleaned, payload_type) = request
```

Comment:
nit: combine these 2 branches into single one

Reply:
actually these seem to be copies of the same code?

Reply:
removed unused